virttest/remote_commander/remote_runner.py

```diff
@@ -295,6 +297,10 @@ def parse_func_name(self, func_name, commander):
         :param func_name: Function name
         :param commander: Where to execute the command (remote or local)
         """
+        for name in func_name:
+            if name == "run_test":
+                # Always run external test in the current process.
+                self.manage = True
```

Comment:
We can run in async mode. But, we do not need it our (spiceqe) case. Async makes all far more complicated (forks, execs, pipes to communicate to child, pipes for STDIO to child, one more messenger for remote_runner and child, another messages..). Another draw back to run in async mode - that we need to have according avocado-test to poll test ( with .wait() ) to wait on master side.